Cubical/ZCohomology/Properties.agda

```diff
@@ -233,23 +233,22 @@ coHomGr≅coHomRedGr : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
 GroupEquiv.eq (coHomGr≅coHomRedGr n A) = isoToEquiv (Iso-coHom-coHomRed n)
 GroupEquiv.isHom (coHomGr≅coHomRedGr n A) = +∙≡+ n
 
-private
-  coHomGroup≡coHomRedGroup' : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
-                          → _ ≡ coHomGroup (suc n) (typ A)
-  coHomGroup≡coHomRedGroup' n A =
-    sym (InducedAbGroupPath (coHomGroup (suc n) (typ A))
-                   (coHomRed (suc n) A , _+ₕ∙_)
-                   (isoToEquiv (invIso (Iso-coHom-coHomRed n)))
-                   (homhelp n A))
-
 coHomRedGroup : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ) → AbGroup {ℓ}
 coHomRedGroup zero A = coHomRedGroupDir zero A
-coHomRedGroup (suc n) A = coHomGroup≡coHomRedGroup' n A i0
+coHomRedGroup (suc n) A =
+  InducedAbGroup (coHomGroup (suc n) (typ A))
+                 (coHomRed (suc n) A , _+ₕ∙_)
+                 (isoToEquiv (invIso (Iso-coHom-coHomRed n)))
+                 (homhelp n A)
 
 abstract
   coHomGroup≡coHomRedGroup : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
                           → coHomRedGroup (suc n) A ≡ coHomGroup (suc n) (typ A)
-  coHomGroup≡coHomRedGroup = coHomGroup≡coHomRedGroup'
+  coHomGroup≡coHomRedGroup n A =
```

Comment:
Wait a minute, the name of this lemma is not the same order as in the type. Please swap the type and remove the `sym`

Reply:
ping @aljungstrom

Reply:
wups, forgot about this. It's fixed now.